packages/ricos-plugins/web/src/editorPlugins.ts

```diff
@@ -61,6 +62,15 @@ export class EditorPlugins implements IEditorPlugins {
     return this.plugins.forEach(plugin => plugin.configure(config));
   }
 
+  getTextButtons() {
+    //maybe use filter class func
+    const textButtons = this.plugins.reduce(
+      (prev, curr) => [...prev, ...(curr.getTextButtons() || [])],
+      []
+    );
+    return textButtons.map(button => new PluginTextButton(button.getButton()));
```

Comment:
not necessary -> new PluginTextButton(button.getButton()) === button
```suggestion
    return textButtons;
```